app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        raise ValueError("Missed some arguments.")
+    cp, file_in, file_out = command.split()
```

Comment:
don't call commad.split() twice
create a separated variable for it and call on demand